backend/src/models/Image.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  url: { primary: true, type: 'string', uri: { allowRelative: true } },
```

Comment:
```suggestion
  url: { primary: true, type: 'string', unique: 'true', uri: { allowRelative: true } },
```

Reply:
this in my mind would make the extra `migration` unnecessary, oder?

Reply:
Primary keys are always unique. The extra migration is an training for us to create migrations when we setup new indices. So, during a helm upgrade the migrations will run but no `init`.